routers/private/serv.go

```diff
@@ -291,9 +275,8 @@ func ServCommand(ctx *context.PrivateContext) {
 
 	// Don't allow pushing if the repo is archived
 	if repoExist && mode > perm.AccessModeRead && repo.IsArchived {
-		ctx.JSON(http.StatusUnauthorized, private.ErrServCommand{
-			Results: results,
-			Err:     fmt.Sprintf("Repo: %s/%s is archived.", results.OwnerName, results.RepoName),
+		ctx.JSON(http.StatusUnauthorized, private.Response{
```

Comment:
Here, a `Forbidden` suddenly makes sense:
```suggestion
		ctx.JSON(http.StatusForbidden, private.Response{
```

Reply:
I don't want to touch old logic too much, let's leave it to the future?